docs/config.md

```diff
@@ -131,3 +135,27 @@ The JSON file has the following fields:
   * `jira`: Optional Jira server and login information
   * `whitesource`: Optional whitesource server authentication information
   * `github_oauth`: Optional project specific GitHub OAuth token
+
+### create findings-[project].yaml files for each project
+
+The `findings-[project].yaml` file (where `[project]` is replaced with the project name used in the config file) contains information used in the generation of the HTML report.  The file contains a list of `findings` with the following fields:
+* `id` Id unique to each finding
+* `priority` Priority of the finding as displayed in the report
+* `matches-license` license name that this finding applies to as it appears in the fossology
```

Comment:
Note that `matches-license` is also optional. Some findings could match just on a path without caring what the license is.

Also note that the license name will be as it appears _after_ the translations applied after retrieving the SPDX file -- e.g. how it appears on the web report, which might be different from how it appears in Fossology.

Reply:
One question - on the translation for the license name - how is that done?  Is there fields in the config file that translate the license names?

Reply:
Restructured this section to (hopefully) make it a bit clearer.

Reply:
For the license name translations, yes, this is tracked in the config.json file.

For a given project, look at the `policies` object, which will define one or more named sub-objects for a policy. Within the policy is a `categories` array, and each category in turn contains a `licenses` array.

Each license object contains a `name` property, which is the name that will appear in the reports; and an `aliases` array of 0+ strings. Each of the strings in an `aliases` array, if there are any, is a value from Fossology's SPDX output that would get "translated" to the `name` value for the reports for that project/policy combination.

Here's an example that might make it easier to understand, this shows one of the objects in a `categories` array:

```
{
    "name": "Attribution",
    "licenses": [
        {
            "name": "BSD (version unspecified)",
            "aliases": [
                "LicenseRef-BSD"
            ]
        },
        {
            "name": "BSD-3-Clause",
            "aliases": []
        }
    }
}, . . .
```

In this example, `BSD` is the license value that the user cleared within Fossology. When Fossology creates the SPDX file, it will use the value `LicenseRef-BSD` as the LicenseConcluded field. And with this configuration, when scaffold ingests that SPDX file, it will translate `LicenseRef-BSD` to `BSD (version unspecified)`, and the latter will be what shows up in reports.

This can be particularly useful for representing complex license expressions. If a file is cleared in Fossology with licenses for e.g. "MIT", "Apache-2.0" and "Dual-license", then in the SPDX file Fossology will correctly represent this as "MIT OR Apache-2.0". But if there's 3 or more entries, then Fossology will output it as e.g. "MIT AND Apache-2.0 AND LicenseRef-Dual-license AND GPL-2.0+". Using the `aliases` to translate that into a different expression can help make it possible to actually convey the real expression in the reports.

Reply:
Thanks @swinslow for the detailed description.  Makes sense.